src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -80,15 +77,27 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
-        private List<string> ConvertToStringLists(ITaskItem[] taskItems)
+        private List<string> GetPackageStringLists(ITaskItem[] taskItems)
         {
             List<string> stringList = new List<string>();
-            foreach (var item in taskItems)
+            foreach (var item in taskItems.Where(i => !i.ItemSpec.Contains("symbols.nupkg")))
```

Comment:
We shouldn't have this extension check in multiple places. We should have it both together outside this function.